### PR TITLE
Make the ILM `freeze` action a no-op

### PR DESCRIPTION
**Pull request by dakrone**
This changes the ILM `freeze` action to not actually freeze the index, instead performing no
operation.

Relates to #70192

### CODE REVIEW

**Comment by joegallo** [comment]
@elasticmachine update branch

File: docs/reference/migration/migrate_8_0/ilm.asciidoc [hunk 1]
```diff
@@ -38,4 +38,16 @@ renamed to `ilm` to match the package rename inside the {es} code.
 Update your workflow and applications to use the `ilm` package in place of
 `indexlifecycle`.
 ====
+
+[[ilm-freeze-zoop]]
```
Comment (joegallo) [inline docs/reference/migration/migrate_8_0/ilm.asciidoc:42]: Is `zoop` here meant to `noop`?

Comment (dakrone) [reply docs/reference/migration/migrate_8_0/ilm.asciidoc:42]: Yep, it definitely is

**Review by joegallo — APPROVED** [review]
🚢

**Comment by dakrone** [comment]
@elasticmachine update branch

**Pull request merged**